docs/source/reference/iterators.rst

```diff
@@ -17,3 +17,18 @@ Chainer provides some iterators that implement typical strategies to create mini
    chainer.iterators.SerialIterator
    chainer.iterators.MultiprocessIterator
    chainer.iterators.MultithreadIterator
+
+
+Order sampler examples
+======================
+
+An Iterator iterates over a dataset according to an order represented by a 1-D array of indices.
+Order samplers are callables that are used by those iterators to generate this array.
+
+
+.. autosummary::
+    :toctree generated/
+    :nosignatures
+
+    chainer.iterators.OrderSampler
+    chainer.iterators.ShuffleOrderSampler
```

Comment:
It looks like these links are broken. Could you take a look?

Reply:
Oh. My bad.